model/browser/events.yaml

```
@@ -0,0 +1,67 @@
+groups:
+  - id: event.browser.page_view
+    stability: development
+    type: event
+    name: browser.page_view
+    brief: >
+      This event represents a browser page load/view.
+    note: >
+      The primary use case is to capture metrics about the number of views of
+      a page. Page view can be a hard page load in a browser as well as virtual
+      navigation in a SPA (single-page application). The event should be
+      collected as soon as possible without waiting for potentially delayed
+      data such as the Navigation timing browser API.
+    body:
+      id: browser.page_view
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: referrer
+          type: string
+          stability: development
+          brief: Referring Page URI (document.referrer) whenever available.
+          requirement_level: recommended
+          examples: ["https://en.wikipedia.org/wiki/Main_Page"]
+        - id: type
+          type: enum
+          members:
+            - id: physical_page
+              value: 0
+              stability: development
+              brief: >
+                Initial page load within the browser which will generally also
+                precede a PageNavigationTiming event.
+            - id: virtual_page
+              value: 1
+              stability: development
+              brief: >
+                This is for Single Page Applications (SPA) where the framework
+                provides the ability to perform client side only page
+                "navigation", the exact definition of what a virtual page
+                change is determined by the SPA and the framework it is using.
+          stability: development
+          requirement_level: required
+          brief: Type of navigation
+          examples: [0, 1]
+        - id: title
+          type: string
+          stability: development
+          requirement_level: recommended
+          brief: Page title DOM property
+          examples: ["Home", "Checkout"]
+        - id: url
+          type: string
+          stability: development
+          requirement_level: required
+          brief: >
+            Full HTTP request URL in the form scheme://host[:port]/path?query[#fragment].
+            Usually the fragment is not transmitted over HTTP, but if it is
+            known, it should be included nevertheless.
+          examples: ["https://en.wikipedia.org/wiki/Main_Page", "https://en.wikipedia.org/wiki/Main_Page#foo"]
```

Should this be a reference to https://github.com/open-telemetry/semantic-conventions/blob/main/model/url/registry.yaml#L39, i.e.

```suggestion
      - ref: url.full
        requirement_level: required
```